containers/fluent-bit/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Stage1: build from source
 FROM ghcr.io/cybozu/ubuntu-dev:22.04 AS builder
 
-ARG FLUENT_BIT_VERSION=3.0.2
+ARG FLUENT_BIT_VERSION=3.1.6
```

Comment:
Recently, a new version, v3.1.7, was released. 
Would you check this version?
https://github.com/fluent/fluent-bit/releases/tag/v3.1.7

Reply:
Thanks. I check the release note and it does not have breaking changes.
So I updated it to 3.1.7.